### PR TITLE
Add aarch64 wheel build support

### PR DESCRIPTION
**Pull request by odidev**
Added aarch64 wheel build support.
Related to https://github.com/joerick/pyinstrument_cext/issues/9, @joerick Could you please review this PR?

### CODE REVIEW

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -5,11 +5,15 @@ on: [push, pull_request]
...
-    name: Build wheels on ${{ matrix.os }}
+    name: Build wheels on ${{matrix.arch}} for ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-18.04, windows-latest, macos-latest]
+        arch: [auto]
+        include:
+          - os: ubuntu-18.04
+            arch: aarch64
```
Comment (joerick) [inline .github/workflows/build_and_test.yml]: Could this be:

```suggestion
        include:
          - os: ubuntu-18.04
            archs: auto
          - os: ubuntu-18.04
            archs: aarch64
          - os: windows-latest
            archs: auto
          - os: macos-latest
            archs: auto universal2
```

Maybe easier just to spell it out... Also, I renamed `arch` -> `archs` to match cibuildwheel's option name.

File: .github/workflows/build_and_test.yml [hunk 2]
```diff
@@ -19,6 +23,10 @@ jobs:
         with:
           python-version: '3.9'
 
+      - name: Set up QEMU
+        if: ${{ matrix.arch == 'aarch64' }}
```
Comment (joerick) [inline .github/workflows/build_and_test.yml]:
```suggestion
        if: ${{ matrix.archs == 'aarch64' }}
```

File: .github/workflows/build_and_test.yml [hunk 3]
```diff
@@ -28,6 +36,7 @@ jobs:
         uses: joerick/cibuildwheel@v1.9.0
         env:
           CIBW_SKIP: pp*
+          CIBW_ARCHS_LINUX: ${{matrix.arch}}
           CIBW_ARCHS_MACOS: auto universal2
```
Comment (joerick) [inline .github/workflows/build_and_test.yml]:
```suggestion
          CIBW_ARCHS: ${{matrix.archs}}
```

**Review by joerick — COMMENTED** [review]
Thanks for the PR @odidev ! I have just a few suggestions.

**Comment by odidev** [comment]
> Thanks for the PR @odidev ! I have just a few suggestions.

Done.

**Comment by joerick** [comment]
Thanks! I'll try to get a release out of this in PyPI. It might also be worth doing the same on pyinstrument proper, since the recently released v4.0, the extension is now part of that project.

**Comment by odidev** [comment]
Thanks @joerick, OK. I will adding pyinstrument in my work queue.

**Pull request merged**